README.md

```diff
@@ -10,6 +10,24 @@ running the [Hastexo
 XBlock](https://github.com/hastexo/hastexo-xblock) in your Tutor Open
 edX platform.
 
+Version compatibility matrix
+----------------------------
```

Comment:
Should we add the XBlock versions to the matrix also?

Reply:
Yeah that's a good idea, for this specific plugin.

Reply:
Yes that's a good idea. I'll add:
```
Lilac: Not Supported
Maple: <= 6.2.0
Nutmeg: >= 7.00 
```
Is that correct?

Reply:
Lilac was supported in Hastexo XBlock. So it should be:
```
Lilac: <=6.0
Maple: >=6.0, <= 6.2.0
Nutmeg: >= 7.00 
```
correct?

Reply:
Well that's kinda moot, since you can't install the XBlock with this plugin in Lilac, anyway. I'd just put "Not supported" in the "Hastexo XBlock version" column too.

Reply:
I think it should be:
```
Lilac: <6.0 
Maple: >=6.0, <7 
Nutmeg: >= 7.00
```